display_list/skia/dl_sk_canvas.h

```diff
@@ -32,7 +32,8 @@ class DlSkCanvasAdapter final : public virtual DlCanvas {
   void Save() override;
   void SaveLayer(const SkRect* bounds,
                  const DlPaint* paint = nullptr,
-                 const DlImageFilter* backdrop = nullptr) override;
+                 const DlImageFilter* backdrop = nullptr,
+                 int64_t backdrop_id = -1) override;
```

Comment:
Do we use this directly (as in on an instance of this class as opposed to casting it to a DlCanvas where the defaults already exist)? Actually we might in the embedders or other platform code.